### DISCUSSION TITLE
Date to find two archives with same name

### DISCUSSION THREAD

**Discussion by jcf278** [Flow-Launcher/Flow.Launcher#1647] [Q&A]
Hi everyone.

There's a way to add a date to search two or more archives with the same name?

The example bellow is about a file I have on two different locations, In one location is newer, but I cannot recognize wich is the right file to select in the bar.

![image](https://user-images.githubusercontent.com/56937020/207611069-f553ad07-d260-498e-bc27-61c4d48b4d8d.png)

**Comment by jjw24** [comment]
Use the 'Preview' feature should allow you to see more. Using Everything would probably allow you to sort or filter by date.

**Comment by jjw24** [comment]
Closing this one, refer to #1648 instead.
